apps/builder/app/builder/features/style-panel/sections/box-shadows/box-shadows.tsx

```diff
@@ -76,9 +78,34 @@ export const Section = (props: SectionProps) => {
           property={property}
           layers={value}
           {...props}
-          renderLayer={(layersProps) => (
-            <BoxShadowLayer key={layersProps.index} {...layersProps} />
-          )}
+          renderLayer={(layersProps) => {
+            return (
+              <ShadowLayer
+                key={layersProps.index}
+                {...layersProps}
+                label={label}
+                tooltip={
+                  <Tooltip
+                    variant="wrapped"
+                    content={
+                      <Text>
+                        Paste a box-shadow CSS code
+                        <br />
```

Comment:
same q, how does it look like without these breaks

Reply:
Without breaks, `jsx` is not preserving the line breaks if the content is less. Update the PR now for the code part alone on tooltip. 👍

Reply:
I don't think there was a need for line break there, not sure if you started it or someone else, but there should be no need for breaks there except the last one after example:<br />

I think you used breaks last time, we just didn't notice

Reply:
@kof updated the PR, it's ready to go now if i get a approval. Meanwhile i was refactoring the whole `layer` management thing in a seperate PR to make it simple while working in a diff branch
https://github.com/webstudio-is/webstudio/tree/refactor-style-panel